src/module/token.ts

```diff
@@ -114,14 +114,34 @@ export class LancerToken extends Token {
       "getOccupiedSpaces is deprecated in favor of the core getOccupiedGridSpaceOffsets",
       { since: 13, until: 14 }
     );
-    return this.document.getOccupiedGridSpaceOffsets()?.map(o => canvas.grid?.getCenterPoint(o));
+    return this.document
+      .getOccupiedGridSpaceOffsets()
+      .map(o => canvas.grid?.getCenterPoint(o))
+      .filter(o => !!o);
+  }
+
+  /**
+   * This exists only to expose protected hover methods from {@link Token}.
+   */
+  onHoverIn(ev: Canvas.Event.Pointer | MouseEvent): void {
```

Comment:
An alternative is `LancerToken["onHoverIn"]`; this is basically an escape hatch for the TypeScript error about protected members.

Also this `Canvas.Event.Pointer | MouseEvent`; does this imply a fix is necessary to fvtt-types?

Reply:
The MouseEvent is just there to make calling the methods easier. Since they were called from native DOM-Events. The canvas pointer event requires a lot more properties to be added, which are not even used in the two token-methods. `_onHoverOut` doesn't even use the event at all. I'd actually argue we can widen the parameter types in the types for these two methods. Yes, they are documented as being full custom canvas pointer events, but they are not used as such.